src/platform/EFR32/ConfigurationManagerImpl.cpp

```diff
@@ -34,9 +34,25 @@ namespace DeviceLayer {
 
 using namespace ::chip::DeviceLayer::Internal;
 
-/** Singleton instance of the ConfigurationManager implementation object.
+/** Singleton pointer to the ConfigurationManager implementation.
```

Comment:
this code is copy & pasted into many places. Could we create a separate .cpp source and symlink that instead? Or  is that not working because header usage?

Reply:
Please, don't create symlinks as they don't work on windows. However, lots of common platform things are put in `src/platform` directory and added to all the platforms in GN.

Reply:
I've split out the singleton implementation into its own file and included it in each platform other than fake/.